backend/tests/helpers/UserHelper.test.js

```diff
@@ -1,40 +1,69 @@
 const { expect } = require( 'chai' );
-const { getUserListFromSessions } = require( '../../src/helpers/UserHelper' );
+const { getUserListFromSessions } = require( '../../src/helpers/helpers' );
+
+const nowTimestamp = Date.now();
+const pastTimeTimestamp = Date.now() - 10000;
 
 const sessionsData = {
 	IbVmSiD_LoULFK2yAAAB: {
 		id: '123',
 		state: 'commenting',
+		joinedAt: nowTimestamp
 	},
 	IbVmSiD_LoUPLNYAAAB: {
 		id: '123',
 		state: 'editing',
+		joinedAt: nowTimestamp
 	},
 	Boc1d1GISKfDnD5zAAA: {
 		id: '123',
 		state: 'away',
+		joinedAt: nowTimestamp
 	},
 	Boc1GUHISK321D5zAAAQ: {
 		id: '123',
 		state: 'viewing',
+		joinedAt: nowTimestamp
 	},
 	Boc1GUHIS423DZNSQERQ: {
 		id: '1234',
 		state: 'viewing',
+		joinedAt: pastTimeTimestamp
 	}
 };
 
+const singleSessionData = {
+	IbVmSiD_LoULFK2yAAAB: {
+		id: '123',
+		state: 'commenting',
+	},
+};
+
 describe( 'getUsers', function() {
-	it( 'Should return list of users', function() {
+	it( 'Should return sorted list of users', function() {
```

Comment:
How the list should be sorted?

Reply:
Ascending. I've added a note in the test description.